core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -2307,6 +2322,14 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.or(OperandTypes.INTEGER_BOOLEAN, OperandTypes.INTEGER),
           SqlFunctionCategory.TIMEDATE);
 
+  @LibraryOperator(libraries = {ORACLE})
```

Comment:
similar as above we already have a fucntion definition called TO_HEX can that not be used?

Reply:
yes sorry, I meant FROM_HEX fucntion. Both these functions convert hexadeimal String to Binary data right?

Reply:
yes both functions representing the same functionality.I have removed HEXTORAW function.Please have a look.